drivers: rtio: workq

FlorianWeber1018 opened the pull request:
Inform the executor of a submissions completion with -ENOMEM if the size of the workq is not big enough.

ubieda reviewed (dismissed):
Thanks!

teburd reviewed (dismissed):
Looks good to me. Probably can remove the asserts if we handle the error path here.

FlorianWeber1018 committed:
drivers: sensors: rtio

Inform the executor of a submissions completion with -ENOMEM
if the size of the workq is not big enough.

Signed-off-by: Florian Weber <Florian.Weber@live.de>

FlorianWeber1018 commented:
asserts are removed now
